airbyte-scheduler/src/main/java/io/airbyte/scheduler/JobSubmitter.java

```
@@ -120,7 +119,8 @@ void submitJob(Job job) {
           }
           trackCompletion(job, output.getStatus());
         })
-        .setOnException(noop -> {
+        .setOnException(e -> {
+          LOGGER.error("Exception thrown in Job Submission: ", e);
```

this is the reason exception in on start and run were getting swallowed.